extensions/amp-access-fewcents/amp-access-fewcents.md

```diff
@@ -0,0 +1,253 @@
+---
```

Comment:
It looks like this doc is not fully finished. Some TODO markers in it, and references to Bento which I'm not sure apply in this case? I'm not sure what is the best practice here. @calebcordry should we wait to add a document until ready, or is there a way to mark it as "draft"?

Reply:
+1 Lets add docs at the very end when everything is fleshed out? Or at least mark it as draft as suggested.

Reply:
I will mark it as draft.

Reply:
@twifkak do you know how to actually mark this md as draft? I am not sure